rewrite-groovy/src/main/java/org/openrewrite/groovy/GroovyTemplate.java

```diff
@@ -99,33 +88,11 @@ public Builder imports(String... fullyQualifiedTypeNames) {
             return this;
         }
 
-        private void validateImport(String typeName) {
-            if (StringUtils.isBlank(typeName)) {
-                throw new IllegalArgumentException("Imports must not be blank");
-            } else if (typeName.startsWith("import ")) {
-                throw new IllegalArgumentException("Imports are expressed as fully-qualified names and should not include an \"import \" prefix");
-            } else if (typeName.endsWith(";") || typeName.endsWith("\n")) {
-                throw new IllegalArgumentException("Imports are expressed as fully-qualified names and should not include a suffixed terminator");
-            }
-        }
-
         Builder parser(GroovyParser.Builder parser) {
```

Comment:
We should probably make this public while we're at it.

```suggestion
        public Builder parser(GroovyParser.Builder parser) {
```